src/main/java/com/google/sps/servlets/SignupMentorServlet.java

```diff
@@ -104,19 +111,50 @@ private void addMentorExperience(String[] experienceTags) {
 
       try {
         // Establish connection to MySQL database.
-        Connection connection = DriverManager.getConnection(Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+        Connection connection = DriverManager.getConnection(
+            Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
 
         // Create the MySQL INSERT prepared statement.
         PreparedStatement preparedStatement = connection.prepareStatement(query);
-        preparedStatement.setInt(1, userId);
-        preparedStatement.setInt(2, Integer.parseInt(tag));
+        preparedStatement.setInt(SqlConstants.MENTOR_EXPERIENCE_INSERT_ID, userId);
+        // Should not be possible for parseInt() to fail because form only allows integer values to
+        // be submitted.
+        preparedStatement.setInt(SqlConstants.MENTOR_EXPERIENCE_INSERT_TAG, Utility.tryParseInt(tag));
         preparedStatement.execute();
         connection.close();
       } catch (SQLException exception) {
         // If the connection or the query don't go through, get the log of the error.
-        Logger logger = Logger.getLogger(MentorSignupServlet.class.getName());
+        Logger logger = Logger.getLogger(SignupMentorServlet.class.getName());
         logger.log(Level.SEVERE, exception.getMessage(), exception);
       }
     }
   }
+
+  /**
+   * Inserts a new entry to MentorEvidence table with default values.
+   */
+  private void addMentorEvidence() {
+    int userId = Utility.getUserId();
+    String paragraph = "";
```

Comment:
Is this ever used?

Reply:
Yes! Right at the end of the doPost() function.

Reply:
I actually meant the 'paragraph' variable!

Reply:
Oh, wow, it's not. Deleted!